e2e/src/HomeFeed.spec.js

```diff
@@ -0,0 +1,48 @@
+import { reloadReactNative } from './utils/retries'
+import { quickOnboarding, sleep, waitForElementId } from './utils/utils'
+
+const jestExpect = require('expect')
+
+beforeAll(async () => {
+  await quickOnboarding()
+})
+
+beforeEach(async () => {
+  await reloadReactNative(false)
```

Comment:
Sorry, I know I've already asked, but can we do without reloading React Native?

Reply:
I reloaded in this case so that the first test that scrolls the transaction feed won't impact the state in the second test that is checking an item in the feed. I'll swap the test order and try without reloading the feed.

Reply:
Removed in https://github.com/valora-inc/wallet/pull/3192/commits/d893332e71e3946ec1209ad84fbec663686d6373!